### PR TITLE
Set default search radius on FL

### PR DESCRIPTION
**Pull request by brangi**
## Description
Issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/17913
 
## Screenshots

Screen Shot 2020-12-29 at 4 02 39 PM

## Acceptance criteria
- [x] Search results returned for zip code are be the same as the results returned for the corresponding city and state 

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/facility-locator/config.js [hunk 1]
```diff
@@ -195,3 +196,16 @@ export const facilityTypesOptions = {
...
+    fbox[0].bbox[1],
+    fbox[0].bbox[0],
+    fbox[0].bbox[3],
+    fbox[0].bbox[2],
+  );
+
+  if (radius < MIN_RADIUS) radius = MIN_RADIUS;
+
+  return radius;
+};
```
Comment (mpelzsherman) [inline src/applications/facility-locator/config.js:211]: Seems like this function belongs in `utils/facilityDistance.js`?

File: src/applications/facility-locator/config.js [hunk 2]
```diff
@@ -195,3 +196,16 @@ export const facilityTypesOptions = {
   [LocationType.CEMETARY]: 'VA cemeteries',
   [LocationType.VET_CENTER]: 'Vet Centers',
 };
+
+export const radiusFromBoundingBox = fbox => {
```
Comment (mpelzsherman) [inline src/applications/facility-locator/config.js:200]: Can we make a unit test for this?

Comment (brangi) [reply src/applications/facility-locator/config.js:200]: Wil do

File: src/applications/facility-locator/config.js (hunk 1 above)
Comment (brangi) [reply src/applications/facility-locator/config.js:211]: Sounds good

File: src/applications/facility-locator/config.js [hunk 3]
```diff
@@ -195,3 +196,16 @@ export const facilityTypesOptions = {
...
+  let radius = distBetween(
+    fbox[0].bbox[1],
+    fbox[0].bbox[0],
+    fbox[0].bbox[3],
+    fbox[0].bbox[2],
+  );
+
+  if (radius < MIN_RADIUS) radius = MIN_RADIUS;
+
+  return radius;
```
Comment (mpelzsherman) [inline src/applications/facility-locator/config.js:210]:
```suggestion
  return Math.max(radius, MIN_RADIUS);
```

**Review by mpelzsherman — APPROVED** [review]

**Pull request merged**